libs/permission-checker/composer.json

```diff
@@ -61,6 +61,6 @@
             "@putenv XDEBUG_MODE=coverage",
             "phpunit --coverage-clover /tmp/build-logs/clover.xml --coverage-xml=/tmp/build-logs/coverage-xml --log-junit=/tmp/build-logs/phpunit.junit.xml"
         ],
-        "infection": "infection --threads=4 --min-covered-msi=90 --coverage=/tmp/build-logs --only-covered"
+        "infection": "infection --threads=4 --min-covered-msi=100 --coverage=/tmp/build-logs --only-covered"
```

Comment:
Zacina mi davat smysl u permissions vyzadovat nulovou toleranci uniklych mutaci. Zase mi to objevilo nejake neotestovane situace.

Reply:
WOW :) Ale jsem rád, že aspoň někdo přišel na chuť infection testům
A současně jsem rád, že jsme to takhle vytáhli ven, protože tohle by se na apce nikdy udělat nedalo.